app/jenkins-for-jira-ui/src/api/getAllJenkinsServers.ts

```diff
@@ -6,7 +6,6 @@ import {
 	AnalyticsTrackEventsEnum
 } from '../common/analytics/analytics-events';
 import { AnalyticsClient } from '../common/analytics/analytics-client';
-import { log } from '../common/logger/client-logger';
```

Comment:
This was added temporarily while testing logging from the client. That's a pipe dream so ripping this out.